cmd/lncli/commands.go

```diff
@@ -101,7 +101,9 @@ func actionDecorator(f func(*cli.Context) error) func(*cli.Context) error {
 			// two commands.
 			if s.Code() == codes.Unimplemented &&
 				(c.Command.Name == "create" ||
-					c.Command.Name == "unlock") {
+					c.Command.Name == "unlock" || 
+					c.Command.Name == "changepassword" ||
+					c.Command.Name == "createwatchonly") {
 				return fmt.Errorf("Wallet is already unlocked")
```

Comment:
new line missing between the conditions and the return statement

Reply:
Why did you ignore this? The existing code didn't confirm to the guidelines, but new code should always try to follow the style rules: https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#additional-style-constraints